src/Tests/RemoteUnitTestExecutor.Host/RemoteUnitTestExecutor.Host.csproj

```diff
@@ -4,6 +4,7 @@
 <Project>
     <PropertyGroup>
         <SubComponent>$(TargetFramework)</SubComponent>
+        <Platforms>x86</Platforms>
```

Comment:
shouldn't this also include at least x64?

Reply:
It looks like this project is only ever used in x86 - see [here](https://github.com/microsoft/CLRInstrumentationEngine/blob/5b401bc8f0860ab527ec9fcd383bb4f01ec4f75a/src/Tests/RemoteUnitTestExecutor/TestEngineBase.cs#L43)

Reply:
The sln file should only be building this project for x86 as well, I'll fix that.